gui/resources.xml

```diff
@@ -210,15 +211,15 @@ Vocabulary
           <!-- Code in document.py takes care of ltr/rtl details -->
           <signal name="activate" handler="undo_cb"/>
         </object>
-        <accelerator key="z"/>
+        <accelerator key="z" modifiers="GDK_CONTROL_MASK"/>
```

Comment:
Yeah, that would probably be a good idea to to have the Z and Y key to be in `gui/document.py` for backwards compatibility sake. Then we could move the Ctrl+Z and Ctrl+Shift Z from `gui/document.py` to here in the `resources.xml` instead.